static/js/add-shows/init.js

```diff
@@ -44,6 +44,26 @@ MEDUSA.addShows.init = function() {
             });
         });
 
+        $('#rootDirs').on('change', function() {
+            if ($('#rootDirs option:selected').length === 0) {
+                $('button[data-add-show]').prop('disabled', true);
+                if (!$('#configure_show_options').is(':checked')) {
+                    $('#configure_show_options').trigger('click');
```

Comment:
Why are you triggering a check when you could just run whatever the click event should?